### DISCUSSION TITLE
Log viewer isn't working on my Livewire Project

### DISCUSSION THREAD

**Discussion by naseerr** [opcodesio/log-viewer#231] [Q&A]
I have downloaded the library and published the assets but it failed to load the log viewer it shows white screen. It shows app.css failed to load. Please refresh the page, re-publish Log Viewer assets, or fix routing for vendor assets. and app.js failed to load. Please refresh the page, re-publish Log Viewer assets, or fix routing for vendor assets. What is the fix?

I'm using 
- My PHP "php": "^7.3|^8.0",
- Laravel Framework 8.83.27
- livewire/livewire v2.12.3

1- Below in the public its showing the app.css and app.js
![public](https://github.com/opcodesio/log-viewer/assets/23665840/a598645d-e955-4317-839c-f7fcb74fd071)

2- In the network console firstly the headers shows 302 found then it shows 404 not found for both files
![headers](https://github.com/opcodesio/log-viewer/assets/23665840/b02b0ecc-9663-4fcd-8e08-b61fbb7be690)

**Comment by arukompas** [comment]
@naseerr hey! 

I can see that the `/vendor/log-viewer/app.css` file above is being redirected to `/en/vendor/log-viewer/app.css`. Do you know why? Do you have certain middleware for multi-language paths?

If so, make sure that the `/vendor/*` paths are ignored by that feature, because these are raw static files served **exactly** from `/vendor/*` folder. The `/vendor/log-viewer/*` files should not be redirected anywhere. Log Viewer itself does not do any redirection, and neither does the default Laravel installation, so something must be configured differently in your app :) 

Let me know how it goes!
